spec/pane-axis-element-spec.coffee

```diff
@@ -0,0 +1,58 @@
+PaneAxisElement = require '../src/pane-axis-element'
+
+describe "PaneResizeHandleElement", ->
+  describe "add and remove", ->
+    class TestItemElement extends HTMLElement
+      createdCallback: ->
+        @classList.add('test-root')
+
+    TestItemElement = document.registerElement 'atom-test-item-element', prototype: TestItemElement.prototype
+
+    [paneAxis, model, Model] = []
+    beforeEach ->
+      # This is a dummy item to prevent panes from being empty on deserialization
+      class Model
```

Comment:
On second thought, I think this might work better as an integration test on `PaneContainer` or even by enhancing existing tests that do splitting to add assertions about the draggable handles between axis children. Also, we need some tests that actually tested the resize functionality.